.buildkite/pipeline.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+steps:
+  - label: ":wave: Greetings"
+    command: "crawl.sh"
+    agents:
+      image: "docker.io/library/hello-world"
```

Comment:
You might want to use the [`bash`](https://hub.docker.com/_/bash) image here, as I believe `hello-world` will just do it's "hello from ..." output and then exit.